docs/RockLib0004.md

```diff
@@ -45,7 +45,7 @@ namespace TestingNamespace
 ```
 
 ### Does not violate
-
+```csharp
```

Comment:
Personally I tend to go with "C#" here instead of "csharp", though I don't think we're consistent on this, and....frankly, it doesn't matter, so I'll just approve.

Reply:
the code highlighter that the docs site uses does not allow `C#` here. Available tags `csharp`,` cs`, `dotnet`